### PR TITLE
[Merged by Bors] - Remove remaining camerapos bindings

### LINKED ISSUE #1706: Alien Cake Addict broken after PBR merge
## Bevy version

45b2db70705da24a89426e6b6e77d603a3983025

## Operating system & version

macOS

## What you did

`cargo run --example alien_cake_addict`

## What actually happened

```
wgpu error: Validation Error

Caused by:
    In Device::create_bind_group
    buffer binding size 64 is less than minimum 80

thread 'main' panicked at 'Handling wgpu errors as fatal by default', /Users/robparrett/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.7.0/src/backend/direct.rs:1896:5
```

## Additional information

~~One~~Two other users in discord reported seeing this as well.

### PR DESCRIPTION
**Pull request by mtsr**
Fixes #1706

@JeanMertz already solved it. I just ran all examples and tests.

### CODE REVIEW

File: crates/bevy_render/src/shader/shader_reflect.rs [hunk 1]
```diff
@@ -379,10 +378,7 @@ mod tests {
                             name: "CameraViewProj".into(),
                             bind_type: BindType::Uniform {
                                 has_dynamic_offset: false,
-                                property: UniformProperty::Struct(vec![
-                                    UniformProperty::Mat4,
-                                    UniformProperty::Vec4
-                                ]),
+                                property: UniformProperty::Struct(vec![UniformProperty::Mat4,]),
```
Comment (bjorn3) [inline crates/bevy_render/src/shader/shader_reflect.rs:381]:
```suggestion
                                property: UniformProperty::Struct(vec![UniformProperty::Mat4]),
```
nit

**Review by alice-i-cecile — APPROVED** [review]

**Comment by cart** [comment]
This one was my fault. Wasn't thorough enough when merging my camera uniform changes with the pbr changes. Thanks!

**Comment by mtsr** [comment]
> This one was my fault. Wasn't thorough enough when merging my camera uniform changes with the pbr changes. Thanks!

And I missed it in review.

**Pull request closed without merging**